src/components/Modal/ExtensionStatusModal.tsx

```
@@ -0,0 +1,300 @@
+                            </p>
+                        </div>
+                    )}
+
+                    <div
+                        className={styles.extensionButtonContainer}
+                        style={{
+                            justifyContent:
+                                data?.allExtensionRequests &&
+                                data.allExtensionRequests.some(
+                                    (request) => request.status === 'PENDING'
+                                )
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)